### ISSUE TITLE
Media: Label Images with Alt Text for Screen Readers

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#41510]
#### Steps to reproduce

1. Open the Calypso Media library using a screen reader
2. Swipe left to right to navigate through images

#### What I expected

I expected the images to be labelled with the Alt text so that a visually-impaired user can identify what the images are.

#### What happened instead

Images are labelled as the file name, which is usually something like IMG042820 and therefore not clear.

**Comment by obenland** [comment]
It looks like it's currently using the image's `alt` value with `title` as a fallback:
https://github.com/Automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/client/my-sites/media-library/list-item-image.jsx#L96

What would be a more appropriate fallback when an image doesn't have `alt` information associated with it?

**Comment by donalirl** [comment]
>What would be a more appropriate fallback when an image doesn't have alt information associated with it?

Title does seem like the most appropriate fallback when an alt text is not set.

Just to clarify, the images I am selecting in that screencast above _do have an alt text associated with them_, but they are not being called out by the screen reader. The file name is being read out by the screen reader, not the title or the alt text which are both set for the images.

**Comment by youbrokesomething** [comment]
The original recording isn't available anymore, however, I did notice that thumbnails of images in Media are reporting alt-labels that have a short description or title. If you open an image and edit the alt text, that alt text does not make it to the media thumbnail images.

**Comment by youbrokesomething** [comment]
If we're picking up any alt text when the image is uploaded, that may be getting represented in the alt-label, but it's not included as alt text in either the thumbnail in Media or in the alt text field when you edit the image.
